### ISSUE TITLE
theme: disableExplicitIndexURLs param is not working as expected

### ISSUE THREAD

**Issue by membrive** [McShelby/hugo-theme-relearn#505]
Hi everyone,

I am trying to configure `disableExplicitIndexURLs` parameter in my new Hugo+Relearn setup. I am using the **latest Relearn theme** version and **Hugo v0.111.3**, deployed in **Netlify**.

My `hugo.toml` looks like the following:

```TOML
baseURL="https://mydomain.com"
canonifyURLs=true
relativeURLs=true
uglyURLs=false

[params]
  disableExplicitIndexURLs = true
```

What I am trying to achieve is urls of this kind:

- `https://mydomain.com/url1/`
- `https://mydomain.com/url2/`
- ...

But in the menu links I see am getting instead:

- `https://mydomain.com/url1/index.html`
- `https://mydomain.com/url2/index.html`
- ...

Following the documentation, I supposed that `disableExplicitIndexURLs=true` would remove those `index.html` from the URL in the menus, but I see them and when I access the links, the index.html is also not removed from the URL.

By other hand, if I manually type `https://mydomain.com/url1/`, it works and the `index.html` is not appended. 

Am I missing something? Or I am facing a real issue?

Thanks in advance.

**Comment by McShelby** [comment]
Fixed in 5.12.4. Thanks for reporting!

**Comment by membrive** [comment]
It works now. Great job @McShelby! Thank you very much.
